ckanext/ckanext-apicatalog_scheming/ckanext/apicatalog_scheming/schemas/dataset.json

```diff
@@ -144,52 +144,112 @@
-      "form_placeholder": "eg. A detailed description",
-      "form_languages": ["fi", "sv", "en"]
+      "label": "Service description",
+      "extra_label": "A general, concise, and easy-to-understand description of the organization",
+      "input_title": ["Description in Finnish", "Description in Swedish", "Description in English"],
+      "validators": "fluent_text mark_as_modified_in_catalog_if_changed",
+      "preset": "fluent_core_title_translated",
+      "required": false,
+      "only_default_lang_required": true,
+      "form_placeholder": "Write subsystem description",
+      "form_languages": ["fi", "sv", "en"],
+      "form_snippet": "fluent_textarea.html",
+      "display_snippet": null,
+      "form_attrs": {}
+    },
+    {
+      "field_name": "access_restriction_level",
+      "label": "Service visibility",
+      "extra_label": "Service visibility",
+      "additional_description": "When service visibility is private, it is only visible to the owner organisation or users from allowed organisations",
+      "validators": "boolean_validator valid_resources",
+      "form_snippet": "radio.html",
+      "display_hr": false,
+      "choices": [
+        {
+          "value": "public",
+          "label": "Public",
+          "checked": true
+        },
+        {
+          "value": "private",
+          "label": "Private",
+          "checked": false
+        }
+      ]
+    },
+    {
+      "field_name": "allowed_organizations",
+      "label": "Allowed Organizations",
+      "classes": ["control-full"],
+      "preset": "apicatalog_organization_string_autocomplete",
+      "form_placeholder": "Allowed Organisations",
+      "validators": "ignore_missing"
     },
     {
       "field_name": "chargeability",
       "label": "Chargeability",
-      "preset": "select",
+      "extra_label": "Other information and settings",
+      "display_hr": true,
+      "validators": "boolean_validator valid_resources",
+      "form_snippet": "select.html",
       "classes": ["control-full"],
       "choices": [
         {
-          "value": "yes",
-          "label": "Yes"
+          "value": "chargeable",
+          "label": "Chargeable"
         },
         {
-          "value": "no",
-          "label": "No"
+          "value": "free_of_charge",
+          "label": "Free of charge"
         },
         {
-          "value": "maybe",
-          "label": "See description"
+          "value": "mentioned_in_service_description",
+          "label": "Mentioned in service description"
         }
       ]
     },
     {
       "field_name": "format",
       "label": "Format",
       "classes": ["control-full"],
-      "preset": "apicatalog_resource_format_autocomplete"
+      "form_snippet": "select.html",
+      "choices": [
+        {
+          "label": "WSDL",
+          "value": "wsdl"
+        },
+        {
+          "label": "OpenAPI JSON",
+          "value": "openapi-json"
+        }
+      ]
+    },
```

Comment:
This still needs to be an autocomplete field, we can have other types than wsdl and openapi as user can add pdf as instructions.

Reply:
Fixed